mongodbatlas_test.go

```diff
@@ -340,6 +388,16 @@ func TestAcceptanceDatabaseUser_UpdateUser_Password(t *testing.T) {
 	assertCredsExists(t, projectID, publicKey, privateKey, dbUser, newPassword, connURL, "")
 }
 
+func assertUsernameTemplate(t testing.TB, username, expectedUsername string) {
```

Comment:
I'm not sure what value this function is providing. As far as I can tell it's just doing a string equality check. What do you think about keeping this vs moving the assertion directly into the test above?

Reply:
Yeah, I had the exact same thought. Initially this test was doing alot more. But then I figured a string equality check is best and simplified it.